src/angular/styles/_typography.scss

```diff
@@ -1764,6 +1764,10 @@ textarea {
     line-height: sbb.pxToRem(20);
   }
 
+  .sbb-checkbox & {
```

Comment:
@mcilurzo Are the text colors for checkbox and radio button intentionally different?

Reply:
I've started a discussion in our Teams channel and temporary removed this code, Later I'll make a new PR for the checkbox and radio button changes.